docusaurus/docs/reactnative/basics/upgrade_helper.mdx

```diff
@@ -32,52 +34,52 @@ Following values in [`ImageGalleryContext`](https://getstream.io/chat/docs/sdk/r
+Offline support is a major opt-in feature introduced in v5 of the Stream Chat React Native SDK.
 
 > Offline support is currently not implemented for Expo package (stream-chat-expo).
 
-Offline storage implementation currently offers following features:
+The offline storage implementation currently offers the following features:
 
-- access to chat when internet is disabled
-- faster loading of chat since chat gets loaded from offline storage first before loading data from network
-- syncing of database using websocket events and sync api.
+- Access to chat when Internet connection is disabled or low.
+- Faster startup times and loading, since initial data is loaded from offline storage before performing any network requests.
+- Syncing of the offline database using WebSocket events and Sync API.
 
-Following features are currently **NOT** implemented. They will be implemented gradually as part of minor releases in v5.
+The following features are currently **NOT** implemented. They will be implemented gradually as part of minor releases in v5.
 
-- optimistically update offline database during chat interactions such as send message, add reactions etc.
-- access to threads in offline mode
-- offline caching of attachment images and user images
+- Optimistically update offline database during chat interactions, such as send message, add reaction, etc.
+- Access to threads in offline mode.
+- Offline caching of attachment images and profile images.
 
-To enable offline support please follow the given steps:
+To enable offline support, please follow the given steps:
 
-- **Upgrade stream-chat dependency (optional)**
+1. **Upgrade stream-chat dependency (optional)**
 
-  If you have installed `stream-chat` dependency explicitly on your application, then upgrade it to v7
+  If you have installed `stream-chat` dependency explicitly on your application, then upgrade it to v7:
 
   ```bash
   yarn add stream-chat@7.0.0
   ```
 
-- **Add `react-native-quick-sqlite` dependency**
+2. **Add `react-native-quick-sqlite` dependency**
 
   ```bash
   yarn add react-native-quick-sqlite
   npx pod-install
   ```
 
-- **Ensure unique instance of Chat component**
+3. **Ensure a unique instance of Chat component**
 
-  Until v4, you could provide separate `Chat` component for each usage of `Channel` component or `ChannelList` component.
-  But from v5, it is necessary that you provide only one instance of `Chat` component within your application.
-  This component needs to be a parent for all the chat related components such as `ChannelList`, `Channel` or `Thread`.
+  Until v4, you could provide a separate `Chat` component for each usage of `Channel` component or `ChannelList` component.
+  But from v5, you must provide only one instance of the `Chat` component within your application.
+  This component needs to be a parent for all the chat-related components such as `ChannelList`, `Channel` or `Thread`.
```

Comment:
@HayesGordon can you please move this part outside of ## Enable Offline Support part (just above the title) because it's relevant also when not enabling offline support

Reply:
Can do!